packages/component-library/src/_Themes/DataVisualizationTheme.js

```diff
@@ -0,0 +1,335 @@
+import { assign } from "lodash";
```

Comment:
We could turn this into a function that returns an object which would allow us to update only a few properties if we want to. The way I see this working is that an object can be taken as a property and we return the default object (the victory theme) with those changes overriding default properties. Kinda like:

```javascript
return {
  ...VictoryTheme,
  ...CustomTheme
}
```

Reply:
@Dianna That sounds good!